modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -2028,6 +2039,24 @@ TEST_F(ROIList, CallInferMultipleTimes)
     validate();
 }
 
+TEST(IEFrameAdapter, blobParams)
+{
+    cv::Mat bgr = cv::Mat::eye(240, 320, CV_8UC3);
+    cv::MediaFrame frame = cv::MediaFrame::Create<TestMediaBGR>(bgr);
+
+    cv::util::any any_params = frame.blobParams();
+    auto params = cv::util::any_cast<std::pair<InferenceEngine::TensorDesc,
+                                               InferenceEngine::ParamMap>>(any_params);
+
+    InferenceEngine::ParamMap pmap({{"HELLO", 42},
+                                    {"COLOR_FORMAT",
+                                     InferenceEngine::ColorFormat::NV12}});
+    InferenceEngine::TensorDesc tdesc;
```

Comment:
So, maybe it makes sense to test with non-empty `TensorDesc`

Reply:
Done